nasa_wildfires/area.py

```diff
@@ -0,0 +1,15 @@
+regions = {
```

Comment:
For consistency with Python traditions, this should have an UPPERCASE name like `REGION_LOOKUP` or `REGION_DICT`.